packages/api/src/services/update_pdf_content.ts

```diff
@@ -40,8 +40,7 @@ export const updateContentForFileItem = async (msg: UpdateContentMessage) => {
         .withRepository(libraryItemRepository)
         .createQueryBuilder('item')
         .innerJoinAndSelect('item.uploadFile', 'file')
-        .where('item.user = :userId', { userId: uploadFile.user.id })
-        .andWhere('file.id = :fileId', { fileId })
+        .where('file.id = :fileId', { fileId })
```

Comment:
the user id check isn't needed because of RLS?

Reply:
> the user id check isn't needed because of RLS?

yes.

also, it could help postgres to use the `upload_file_id` index when issuing the query